db/migrate/20190429030814_create_pieces.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePieces < ActiveRecord::Migration[5.2]
```

Comment:
How were you thinking of connecting the pieces to the game/keeping track of the individual pieces for a game? I think it might be easier to incorporate the pieces/board into the game table.